packages/skipper/tests/FileSystem/PathNormalizerTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\Skipper\Tests\FileSystem;
+
+use Iterator;
+use Symplify\PackageBuilder\Testing\AbstractKernelTestCase;
+use Symplify\Skipper\FileSystem\PathNormalizer;
+
+final class PathNormalizerTest extends AbstractKernelTestCase
+{
+
+    /**
+     * @dataProvider providePaths
+     */
+    public function testPaths(string $path, string $expected): void
+    {
+        $pathNormalizer = new PathNormalizer();
```

Comment:
This should be fetched from container, see other tests.

Reply:
Still trying to get my head around how this would look in this case.

Reply:
I have a feeling that i'm added way to many files, there must be something I don't understand. It's still not running green.